SWLOR.Game.Server/Feature/StatusEffectDefinition/BattleInsightStatusEffectDefinition.cs

```diff
@@ -25,8 +25,8 @@ private void BattleInsight1(StatusEffectBuilder builder)
                 .CannotReplace(StatusEffectType.BattleInsight2)
                 .TickAction((source, target, effectData) =>
                 {
-                    var effect = EffectAttackDecrease(5);
-                    effect = EffectLinkEffects(effect, EffectACDecrease(5));
+                    var effect = EffectAttackDecrease(15);
```

Comment:
These numbers are far higher than I suspect you intended. First of all, there's a cap of 20 by default in NWN. This could be raised with a new setting on the server but I'm like 99% sure you didn't expect to do this.

An attack decrease of 1 actually adjusts accuracy by 5. This happens in the Stat.cs class in the CalculateEffectAccuracy[Native] method. You're reducing accuracy by 75 which will more than likely drop lower level players into the negatives here. 

This is also the case for the AC adjustments below. I don't know what the exact numbers should be, but I'm confident these aren't the right ones.